src/main/java/uk/gov/register/db/RecordQueryDAO.java

```diff
@@ -21,26 +21,25 @@
 import java.util.Optional;
 
 public abstract class RecordQueryDAO {
-
     private static final ObjectMapper objectMapper = Jackson.newObjectMapper();
 
     @SqlQuery("SELECT count FROM total_records")
     public abstract int getTotalRecords();
 
-    @SqlQuery("select entry_number, timestamp, e.sha256hex as sha256hex, key, content from entry e, item i where e.sha256hex=i.sha256hex and e.entry_number = (select entry_number from current_keys where current_keys.key=:key)")
+    @SqlQuery("select e.entry_number, array_agg(ei.sha256hex) as sha256hex, e.timestamp, e.key, array_agg(i.content) as content from entry e join entry_item ei on ei.entry_number = e.entry_number and e.entry_number = (select entry_number from current_keys where current_keys.key=:key) join item i on i.sha256hex = ei.sha256hex group by e.entry_number")
     @SingleValueResult(Record.class)
     @RegisterMapper(RecordMapper.class)
     public abstract Optional<Record> findByPrimaryKey(@Bind("key") String key);
 
-    @SqlQuery("select entry.entry_number, timestamp, entry.sha256hex as sha256hex, entry.key, content from item, entry, current_keys where current_keys.entry_number = entry.entry_number and item.sha256hex=entry.sha256hex order by entry.entry_number desc limit :limit offset :offset")
+    @SqlQuery("select e.entry_number, array_agg(ei.sha256hex) as sha256hex, e.timestamp, e.key, array_agg(i.content) as content from entry e join entry_item ei on ei.entry_number = e.entry_number join item i on i.sha256hex = ei.sha256hex join current_keys ck on ck.entry_number = e.entry_number group by e.entry_number order by e.entry_number desc")
```

Comment:
This must have dropped off when changing the queries back, thanks. Pushing shortly.